onadata/apps/logger/models/submission_review.py

```diff
@@ -42,6 +43,9 @@ class SubmissionReview(models.Model):
         choices=STATUS_CHOICES,
         default=PENDING,
         db_index=True)
+    deleted_at = models.DateTimeField(null=True, default=None)
+    deleted_by = models.ForeignKey(
```

Comment:
lets set on_delete

Reply:
I mean the on_delete parameter for the foreign key field should be set.  One reason being that this will be required when we move to Django 2.0

Reply:
Added in latest commits